languages.json

```
@@ -1148,6 +1148,13 @@
       "multi_line_comments": [["{-", "-}"]],
       "extensions": ["purs"]
     },
+    "Pyret": {
+      "line_comment": ["#"],
+      "multi_line_comments": ["#|", "|#"],
```

```suggestion
      "multi_line_comments": [["#|", "|#"]],
```
I see it now: this should be a list of lists.